user/utils.py

```diff
@@ -0,0 +1,38 @@
+import jwt
+import re
+
+from django.http     import JsonResponse
+
+from config.settings import SECRET_KEY
+from user.models     import User
+from wecha_settings  import TOKEN_ALGORITHM
+
+
+def password_validation(password):
+
+    # 문자 최소 1개 포함, 숫자 최소 1개 포함, 특수문자 최소 1개 포함, 비밀번호 최소 6자리 이상
+    validation_re = re.compile(r'^(?=.*[a-zA-Z])(?=.*[0-9])(?=.*[~!@#$%^&*]).{6,}')
+    if validation_re.match(password):
+        return True
+    else:
+        return False
```

Comment:
```suggestion
    return validation_re.match(password)
```
리턴 하는 값이 boolean일 경우, 그 자체를 리턴하는 것이 보다 파이썬답습니다